### PR TITLE
Rest api for registering accounts and querying statuses

### PR DESCRIPTION
**Pull request by Mingela**

### CODE REVIEW

File: deploy/docker-compose.yml [hunk 1]
```diff
@@ -6,16 +6,21 @@ services:
     build:
       context: ../
       dockerfile: Dockerfile
+    ports:
+      - "0.0.0.0:8080:8080"
```
Comment (x3medima17) [inline deploy/docker-compose.yml]: why do we need 0.0.0.0 ?

Comment (Mingela) [reply deploy/docker-compose.yml]: Sure it can be omitted, done

File: brvs-core/src/main/java/iroha/validation/Application.java [hunk 2]
```diff
@@ -1,18 +1,38 @@
...
-import org.springframework.context.annotation.ComponentScan;
+import iroha.validation.transactions.storage.TransactionVerdictStorage;
+import java.net.URI;
+import org.glassfish.hk2.utilities.binding.AbstractBinder;
+import org.glassfish.jersey.grizzly2.httpserver.GrizzlyHttpServerFactory;
+import org.glassfish.jersey.server.ResourceConfig;
+import org.springframework.context.support.AbstractApplicationContext;
 import org.springframework.context.support.FileSystemXmlApplicationContext;
 
-@ComponentScan("iroha.validation")
```
Comment (dolgopolovwork) [inline brvs-core/src/main/java/iroha/validation/Application.java]: Why do you remove all Spring annotations?

Comment (Mingela) [reply brvs-core/src/main/java/iroha/validation/Application.java]: they seem to be redundant in case of xml configuration

**Pull request merged**